client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -467,7 +467,7 @@
             link: function(scope) {
                 scope.data = null;
                 scope.error = null;
-                scope.type = scope.item.type || scope.item.itemClass.split(':')[1];
```

Comment:
Yes, the latter is exactly what I have done, more clear IMO where the type comes from.

I won't do any changes then, thanks.